### PR TITLE
Adds uui-box examples

### LINKED ISSUE #32: Implement the UmbBox section view
Current implementation is [here](https://github.com/umbraco/UI-Examples/tree/dev/v14/src/Our.Umbraco.UiExamples/content/App_Plugins/uiexamples/umbbox).

image

### PR DESCRIPTION
**Pull request by krebil**
I've tried my hand at implementing https://github.com/umbraco/UI-Examples/issues/32

As umb-box is not a thing anymore I've added examples for uui-box instead. I'm not sure if there is enough descriptive text or not. 

Let me know about any adjustments you would like me to make.

fixes https://github.com/umbraco/UI-Examples/issues/32

### CODE REVIEW

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                ${this.renderHeaderActionsSection()}
+            </div>
+        `
+    }
+
+    renderIntroSection() {
+        return html`
+            <uui-box class="red-outline" .headline="test">
+                <h3 slot="headline" class="blue-outline spacing headline">
+                    The uui-box element
```
Comment (Rockerby) [inline src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: Consider updating this with the same text as in the HTML example code ("this is a headline") to make the example directly tie into what the user sees

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+                </h3>
+                <div slot="header" class="header purple-outline spacing">
+                    Other header content
```
Comment (Rockerby) [inline src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: Consider updating this with the same text as in the HTML example code ("this is a header") to make the example directly tie into what the user sees

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+                        <uui-icon name="info"></uui-icon>
+                        View the Storybook library
+                    </uui-button>
+                </div>
+                <slot>
+                    <div class="green-outline spacing">
+                        <umb-code-block language="HTML">${this.renderBoxCodeExample()}</umb-code-block>
+                        <div>
+                            <p>
+                                The uui-box has largely replaces the umb-box element form previous versions of Umbraco.
```
Comment (Rockerby) [inline src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: Typo; `form` -> `from`

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+        <uui-box>
+            <h3 slot="header">
+                The uui-box header slot
+            </h3>
+            <slot>
+                <div>
+                    <umb-code-block language="HTML">${this.renderHeaderSlotCodeExample()}</umb-code-block>
+                    <p>
+                        The header slot is optional and is used to display additional content in the header of the box. 
+                    </p>
```
Comment (Rockerby) [inline src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: The header slot can also be added via the `header` attribute (like `headline`). Can we add an example of it here too please?

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+            </slot>
+        </uui-box>
+        `;
+    }
+
+    renderBoxCodeExample() {
+        return html`
+&lt;uui-box&gt;
+    &lt;h3 slot=&quot;headline&quot;&gt;
+        this is a title
```
Comment (Rockerby) [inline src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: Consider rewording to `this is a headline`

**Review by Rockerby — CHANGES_REQUESTED** [review]
Hey there @krebil ! Thanks for your contribution to the package 🙌

Great work on this, I think the new page showcases the UUI Box component well. I've added a few comments for you to take a look over before we get this merged in.

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts (hunk 4 above)
Comment (krebil) [reply src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: Actually the only properties on uui-box is headline and headline variant, at least according to the storybook https://uui.umbraco.com/?path=/docs/uui-box--docs

**Comment by krebil** [comment]
@Rockerby I've updated the text and corrected typos based on your comments. I haven't been able to find any documentation on uui-box with a header property and I can't get it to work when i try myself either. So I haven't added an example for that just yet.

File: src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts (hunk 4 above)
Comment (Rockerby) [reply src/Our.Umbraco.UiExamples.v14/src/scripts/box/box-dashboard.ts]: Very correct! Apologies, I was convinced that was an attribute tag as well 😊

**Comment by Rockerby** [comment]
Thanks @krebil . That looks spot on, changes checked and merged #h5yr!

**Pull request merged**